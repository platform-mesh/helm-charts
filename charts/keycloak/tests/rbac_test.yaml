suite: rbac
templates:
  - rbac.yaml
release:
  namespace: openmfp-system
values:
  - ../test-values.yaml
tests:
  - it: matches the snapshot
    asserts:
      - matchSnapshot: {}