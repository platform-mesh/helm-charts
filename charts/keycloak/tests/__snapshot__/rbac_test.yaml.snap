matches the snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: Role
    metadata:
      name: secret-creator
      namespace: openmfp-system
    rules:
      - apiGroups:
          - ""
        resources:
          - secrets
        verbs:
          - create
  2: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: RoleBinding
    metadata:
      name: secret-creator-binding
      namespace: openmfp-system
    roleRef:
      apiGroup: rbac.authorization.k8s.io
      kind: Role
      name: secret-creator
    subjects:
      - kind: ServiceAccount
        name: keycloak-client-creation
        namespace: openmfp-system
